2023-10-13/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+)
+
+func appendEl(s string) string {
+	return s + "el"
+}
+
+func main() {
+	fmt.Println(appendEl("Jo"))
```

Comment:
So here we use the fmt.Println function whichin we call to our previously defined append function to add "el" after our defined variable "Jo" which will result in "Joel"